packages/firebase_core_tizen/lib/firebase_core_tizen.dart

```diff
@@ -0,0 +1,99 @@
+// Copyright 2021 Invertase Limited. All rights reserved.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file.
+
+library firebase_core_desktop;
+
+import 'package:firebase_core_dart/firebase_core_dart.dart' as core_dart;
+import 'package:firebase_core_platform_interface/firebase_core_platform_interface.dart';
+
+part 'firebase_app_desktop.dart';
+
+/// Desktop implementation of FirebaseCore for managing Firebase app
+/// instances.
+class FirebaseCore extends FirebasePlatform {
+  /// Called by PluginRegistry to register this plugin as the implementation for Desktop
+  static void registerWith() {
+    FirebasePlatform.instance = FirebaseCore();
+  }
+
+  // ignore: public_member_api_docs
+  static void register() => registerWith();
```

Comment:
Won't flutter-tizen tool take care of `registerWith`? I believe `register` is only used in flutter tool.

Reply:
It would be nice if `registerWith()` was called but it's not. I don't know why. Without `register()` build fails with:
```
tizen/flutter/generated_main.dart:20:18: Error: Member not found: 'FirebaseCore.register'.
    FirebaseCore.register();
                 ^^^^^^^^
```

Reply:
@HakkyuKim It's the opposite. `registerWith` is used by the flutter_tools for its desktop plugins and we use only `register` for our Dart plugins.

Reply:
Ah right, my mistake!